src/lib.rs

```diff
@@ -347,7 +347,7 @@
 //! let threads = 2;
 //! // NB u64 for k<=31, u128 for k<=63
 //! let merged_dict =
-//!     build_and_merge::<u64>(&input_files, k, rc, &quality, threads);
+//!     build_and_merge::<u64>(&input_files, k, rc, &quality, threads, None);
```

Comment:
Clearer to do what you've done on l367 and set the value with its own variable above so we can see what it is named